packages/shared/providers.yaml

```diff
@@ -1161,6 +1161,15 @@ insightly:
             method: GET
             endpoint: /v3.1/Contacts
     docs: https://docs.nango.dev/integrations/all/insightly
+instantly:
+    categories:
+        - marketing
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.instantly.ai/
+        headers:
```

Comment:
Could you test that it works to insert the API key in the headers? 

In the docs, they use a [query param for GET](https://developer.instantly.ai/campaign-1/get-campaign-accounts), and a [body param for POST](https://developer.instantly.ai/campaign-1/add-sending-account-to-campaign).

Reply:
We use it locally in headers, will confirm for you though.

Reply:
Actually you're correct, I was confused with another one of our integrations.